src/Contexts/CartContext/cart-context.js

```diff
@@ -1,15 +1,37 @@
+import axios from "axios";
 import { createContext, useContext, useEffect, useReducer } from "react";
 import { cartReducer } from "../../Reducers";
+import { useAuth } from "../AuthContext/auth-context";
 
 const CartContext = createContext();
 
 const CartProvider = ({ children }) => {
+  const { state: authState } = useAuth();
+
   const initialState = {
     cartList: [],
   };
 
   const [state, dispatch] = useReducer(cartReducer, initialState);
 
+  useEffect(() => {
+    async function getCartItems() {
+      try {
+        const responce = await axios.get("/api/user/cart", {
+          headers: {
+            authorization: authState.token,
+          },
+        });
```

Comment:
Suggestion : You should use ES6 arrow function syntax , so habit goes to all future projects.